ansible_community_steering_committee.rst

```diff
@@ -70,6 +70,38 @@ Depending on a topic you want to discuss with the Community and the Committee, a
 * `Ansible Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
 * `Ansible Collection Inclusion Checklist <https://github.com/ansible-collections/overview/blob/main/collection_checklist.md>`_.
 
+Community Topics Workflow
+-------------------------
+
+The Committee uses the `Community-topics workflow <https://github.com/ansible-community/community-topics/blob/main/community_topics_workflow.md>`_ to asynchronously discuss and vote on the `community-topics <https://github.com/ansible-community/community-topics/issues>`_.
+
+The quorum, the minimum number of Committee members who must vote on a topic in order for a decision to be officially made, is half of the whole number of the Committee members. 
+
+If the quorum number contains a fractional part, it is rounded using standard rules for rounding whole numbers. For example, if there are thirteen members currently in the committee, the quorum will be seven.
+
+In case of equal numbers of votes for and against a topic, the chairperson's vote will be used as a casting vote to resolve the deadlock. For example, if there are six votes for and six votes against a topic, and the chairperson's vote is among those six which are for the topic, the final decision will be positive. If the chairperson has not voted yet, other members ask them to vote.
```

Comment:
Should we add something for the case of votes with more than two options? Like: the winning option must have at least half of the SC votes?

Reply:
Sounds sensible to me! if you suggest a wording, it would be great

Reply:
Maybe something like:
```suggestion
Votes must always have "no change" as an option.

In case of equal numbers of votes for and against a topic, the chairperson's vote will be used as a casting vote to resolve the deadlock. For example, if there are six votes for and six votes against a topic, and the chairperson's vote is among those six which are for the topic, the final decision will be positive. If the chairperson has not voted yet, other members ask them to vote.

For votes with more than two options, one choice must have at least half of the votes. If two choices happen to both have half of the votes, the chairperson's vote will be used as a casting vote. If no choice has at least half of the votes, the vote choices have to be adjusted so that a majority can be found for a choice in a new vote.
```

Reply:
I like it:) Thanks! committed